### ISSUE TITLE
Training the yolo7 detector crashes in the middle of the training

### ISSUE THREAD

**Issue by YHassan002** [deyiwang89/pytorch-yolov7-deepsort#4]
The error is because, in the "yolo_training.py" inside the "nets" directory inside "build_targets" method, there is a line where you are concatenating the matching_bs[num_layer] tensors. and this line is throwing an error that "cannot process empty tensor"
Any idea how to work around this issue?

for i in range(num_layer):
              matching_bs[i]      = torch.cat(matching_bs[i], dim=0) ## --> The error starts here!
              matching_as[i]      = torch.cat(matching_as[i], dim=0)
              matching_gjs[i]     = torch.cat(matching_gjs[i], dim=0)
              matching_gis[i]     = torch.cat(matching_gis[i], dim=0)
              matching_targets[i] = torch.cat(matching_targets[i], dim=0)
              matching_anchs[i]   = torch.cat(matching_anchs[i], dim=0)
